docs/reductions.md

```diff
@@ -13,7 +13,7 @@ The following distributed program computes the sum from 0 to 999 in `sum_buf` us
 celerity::distr_queue q;
 celerity::buffer<size_t, 1> sum_buf{{1}};
 q.submit([=](celerity::handler& cgh) {
-    auto rd = celerity::reduction(sum_buf, cgh, cl::sycl::plus<size_t>{},
+    auto rd = celerity::reduction(sum_buf, cgh, sycl::plus<size_t>{},
```

Comment:
Not all SYCL implementations (e.g. hipSYCL) expose the `sycl::` namespace when included through the legacy `<CL/sycl.hpp>` header. However we should probably provide an alias in that case - I will create a PR for it.